manifest_staging/charts/eraser/values.yaml

```diff
@@ -97,6 +98,7 @@ deploy:
     tag: "v1.4.0-beta.0"
   additionalArgs: []
   priorityClassName: ""
+  additionalPodLabels: {}
```

Comment:
why do we need additionalPodLabels here?

Reply:
The goal is to be able to set additionalPodLabels on every pod. So we have to handle two parts:
1.) The pod labels for the eraser-controller-manager pod. We define these in deploy.additionalPodLabels, because they are used during the deployment of the controller.
2.) The pod labels for the pods that the controller creates at runtime. We define these in runtimeConfig.manager.additionalPodLabels, because they are used during runtime of the controller.

Reply:
got it, thanks for clarifying!